src/DynamoUtilities/DynamoFeatureFlagsManager.cs

```diff
@@ -20,9 +21,21 @@ namespace DynamoUtilities
     /// </summary>
     internal class DynamoFeatureFlagsManager : CLIWrapper
     {
+        // Utility class that supports mocking during tests
+        class FFlags : IFFlags
+        {
+            T IFFlags.CheckFeatureFlag<T>(DynamoFeatureFlagsManager mgr, string featureFlagKey, T defaultval)
+            {
+                return mgr.CheckFeatureFlagInternal(featureFlagKey, defaultval);
+            }
+        }
+
+        // Useful for mocking in tests
+        internal IFFlags flags { get; set; } = new FFlags();
```

Comment:
I may have tunnel vision, but I could not think of anything else to easily enable mocking.
I seem to need mocking because I am using the CheckFeatureFlag function in the PackageLoader

Reply:
@mjkkirschner any other options that you see?

Reply:
The feature flags cli already returns a set of static test flags in test mode - is that sufficient?